### DISCUSSION TITLE
[Feature Request] World Clock - Display time zones in tooltip

### DISCUSSION THREAD

**Discussion by waenhill** [lxqt/lxqt#1886] [Ideas]
_Copy pasted from there as requested by @tsujan : https://github.com/lxqt/lxqt-panel/issues/1511_

Hello everyone,
I would like to summit an idea for the World clock widget.
I have added several time zones in the tab "Time zones" of that widget, but to be honest I don't understant how it is used...
I can only see my default time zone.

There is an option to display a tooltip on the widget, but it is giving only my default date and time.
So could it be feasable to display all time zones (Custom names if available otherwise IANA id) in the tooltip of the widget?

For instance with two Time zones:
```
India: 11/12/2020 14:30
France: 11/12/2020 10:00
```

Please let me know your comment.
Thanks you in advance!

**Comment by tsujan** [comment]
First, thank you for using Discussions!

World clock already has a popup that shows all your chosen time zones. It's shown by middle clicking and persists until you click somewhere. So, there's no need to a tooltip, which would disappear quickly.

**Reply by waenhill** [reply]
Ohh ok great! Thank you very much
Sorry I feal stupid for not trying the middle click. It is displaying all informations that I need, and even following my custimized format !!

Maybe, can I suggest to display information about that functionnality somewhere?
For instance on the top of "Time zones" tab, or in the tooltip, like
`(middle click for timezones)`

?

**Reply by tsujan** [reply]
> Maybe, can I suggest to display information about that functionnality somewhere?

Yes, I agree that it should be displayed somewhere. There's no reason that the user should think about middle clicking.

**Reply by tsujan** [reply]
BTW, your first comment made me find a bug in time zones of world clock. Will open an issue for it and make a PR to fix it soon.
